README.md

```diff
@@ -131,11 +131,15 @@ Usage: rehearsal graph [options] [srcPath]
 produces the migration order of files given a source path
 
 Arguments:
-  srcPath                  path to a directory or file (default: current working directory)
+  srcPath                  path to a directory or file (default: process.cwd())
```

Comment:
Restore this, I think `saying current working directory` makes sense. Saying process.cwd() implies node knowledge that some people may not have.